app/jobs/verify_linkbacks_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class VerifyLinkbacksJob < ApplicationJob
+  queue_as :default
+
+  retry_on ActiveRecord::RecordNotFound, wait: :exponentially_longer, attempts: 3
+
+  def perform(rubygem_id)
+    linkset = Rubygem.with_versions.find(rubygem_id).linkset
+    gem_name = linkset.rubygem.name
```

Comment:
```suggestion
    rubygem = Rubygem.with_versions.find(rubygem_id)
    linkset = rubygem.linkset
    gem_name = rubygem.name
```

Reply:
Neat, that's what I was thinking to make Code Climate happy! :D